smart-py/marketplace.py

```diff
@@ -6,8 +9,11 @@ def __init__(self, objkt, metadata, manager, fee):
             manager = manager,
             swaps = sp.big_map(tkey=sp.TNat, tvalue=sp.TRecord(issuer=sp.TAddress, objkt_amount=sp.TNat, objkt_id=sp.TNat, xtz_per_objkt=sp.TMutez, royalties=sp.TNat, creator=sp.TAddress)),
             counter = 500000,
-            fee = fee
-            )
+            fee = fee,
+            fee_account = manager,
+            secondary_accounts = sp.set([manager]),
```

Comment:
Since the initial contract manager and then possible subsequent updates to the manager is also part of this set, does the name "secondary_accounts" still make sense?

Reply:
The manager could also change the manager with the update_manager entrypoint. It could be useful that he is in the secondary_accounts in case that the new manager has a "bus accident". In that case the old manager can take control again.
But note that there is a remove_secondary_account entrypoint, so the new manager could remove accounts added by the previous manager.

Reply:
But I have no problem to change the name to a more appropriate name :)